udata/core/dataset/forms.py

```diff
@@ -104,6 +104,10 @@ class ResourceForm(BaseResourceForm):
     id = fields.UUIDField()
 
 
+class ResourceFormWithoutId(BaseResourceForm):
```

Comment:
I'm confused we use ResourceFormWithoutId in DatasetForm?

Reply:
Because otherwise we don't have the ID for the reorder for exemple…